ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/style/StyleFactory.kt

```diff
@@ -424,6 +424,7 @@ internal class StyleFactory(
             ButtonComponentStyle(
                 stackComponentStyle = stack,
                 action = action,
+                transition = component.transition,
```

Comment:
I guess we don't need to do any processing on the transition, so this seems good 👍 (I was just investigating whether we needed to do any transformations, same way we do for actions.